System-Automation-Scripts/Email_Client/README.md

```diff
@@ -0,0 +1,2 @@
+# Email Client using Python for automating email shooting process
```

Comment:
I will recommend you to elaborate the installation process briefly along with this in README file.

Rest, LGTM :+1:

Reply:
Hi @Neilblaze, I have updated the changes as advised. Also, thank you very much for your kind words!

Reply:
@Neilblaze as all the required modules are part of Python Standard Libraries, nothing in particular is required in terms of set-up as long as the user has Python installed on his/her machine :)